requirements/requirements.txt

```diff
@@ -26,7 +26,6 @@ gunicorn==18.0
 lxml==3.4.4
 kafka-python==0.9.4
 mock==0.8.0
```

Comment:
[corehq/apps/accounting/generator.py](https://github.com/dimagi/commcare-hq/blob/3635a60/corehq/apps/accounting/generator.py) is only used by tests, right? If that's true then this can probably moved into test-requirements.txt as long as the import in models.py is also removed.

I'm curious why we have test modules like this that do not live in a `tests` package?